policyengine_us/parameters/gov/states/nc/tax/income/deductions/itemized/mortgage_limit.yaml

```diff
@@ -0,0 +1,11 @@
+description: North Carolina Max amount of Itemized Deductions (Mortgage Interest).
+values:
+  2023-01-01: 10_000
```

Comment:
I though this should be 20_000?

Reply:
"the sum of qualified home mortgage interest and real estate property taxes may not exceed $20,000", plus " an individual who files a North Carolina joint return with a spouse, a single return, or a return as head of household may not deduct more than $10,000 of real estate taxes paid or accrued for the taxable year as a North Carolina itemized deduction." Which leaves mortgage interest deduction with a max of 10,000?